BookStoreProd/src/main/java/mate/academy/bookstoreprod/dto/BookDto.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookstoreprod.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
```

Comment:
you can use `@Data` here
```suggestion
@Data
```